systemtest/src/main/java/io/strimzi/systemtest/performance/report/parser/PerformanceMetricsParser.java

```diff
@@ -0,0 +1,392 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.performance.report.parser;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import io.strimzi.api.kafka.model.common.ContainerEnvVar;
+import io.strimzi.api.kafka.model.kafka.KafkaSpec;
+import io.strimzi.systemtest.performance.PerformanceConstants;
+import io.strimzi.test.TestUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+// TODO: Each component should have it's own parser I would say (different metrics)
+// TODO: We should have some base/common parser on top of the hierarchy to share common performance metrics
```

Comment:
Are those still relevant?

Reply:
No, it's not :) I have implemented such logic now.. :)  thanks